TimoCloud-API/pom.xml

```diff
@@ -7,7 +7,7 @@
     <parent>
         <artifactId>TimoCloud</artifactId>
         <groupId>cloud.timo.TimoCloud</groupId>
-        <version>6.0-beta-1.4</version>
+        <version>6.0-beta-1.5</version>
```

Comment:
Why did you change the version again? :-)

Reply:
Lasergottheute um 16:31 Uhr
huch
musst dein commit wohl auf 1.5 updaten :stuck_out_tongue:
oder 1.4.1

Reply:
weil es ja schon die 1.4 gibt :)